src/index.js

```diff
@@ -1,7 +1,7 @@
 import fs from 'fs-extra';
 import path from 'path';
 import filedirname from 'filedirname';
-const [__dirname] = filedirname(new Error());
+const [_, __dirname] = filedirname(new Error());
```

Comment:
Oh, `__dirname` was assigned the value of `__filename` and that's why the paths got messed up?

Reply:
precisely!

Reply:
Alright, with that being resolved, we are probably finally ready for merging the PR. right?